cps/templates/listenmp3.html

```diff
@@ -0,0 +1,260 @@
+            </span>
+            {% endfor %} {% endif %} {% endfor %}
+        </p>
+      </div>
+      {% endif %}
+
+      <h3>{{_('Description:')}}</h3>
+      {{entry.comments[0].text|safe}}
+    </div>
+
+    <div class="sm2-bar-ui compact flat">
+
+      <div class="bd sm2-main-controls">
+
+        <div class="sm2-inline-texture"></div>
+        <div class="sm2-inline-gradient"></div>
+
+        <div class="sm2-inline-element sm2-button-element">
+          <div class="sm2-button-bd">
+            <a href="#play" class="sm2-inline-button sm2-icon-play-pause">Play / pause</a>
+          </div>
+        </div>
+
+        <div class="sm2-inline-element sm2-inline-status">
+
+          <div class="sm2-playlist">
+            <div class="sm2-playlist-target">
+              <!-- playlist <ul> + <li> markup will be injected here -->
+              <!-- if you want default / non-JS content, you can put that here. -->
+              <noscript>
+                <p>JavaScript is required.</p>
+              </noscript>
+            </div>
+          </div>
+
+          <div class="sm2-progress">
+            <div class="sm2-row">
+              <div class="sm2-inline-time">0:00</div>
+              <div class="sm2-progress-bd">
+                <div class="sm2-progress-track">
+                  <div class="sm2-progress-bar"></div>
+                  <div class="sm2-progress-ball">
+                    <div class="icon-overlay"></div>
+                  </div>
+                </div>
+              </div>
+              <div class="sm2-inline-duration">0:00</div>
+            </div>
+          </div>
+
+        </div>
+
+        <div class="sm2-inline-element sm2-button-element sm2-volume">
+          <div class="sm2-button-bd">
+            <span class="sm2-inline-button sm2-volume-control volume-shade"></span>
+            <a href="#volume" class="sm2-inline-button sm2-volume-control">volume</a>
+          </div>
+        </div>
+
+      </div>
+
+      <div class="bd sm2-playlist-drawer sm2-element">
+
+        <div class="sm2-inline-texture">
+          <div class="sm2-box-shadow"></div>
+        </div>
+
+        <!-- playlist content is mirrored here -->
+
+        <div class="sm2-playlist-wrapper">
+          <ul class="sm2-playlist-bd">
+            <li>
+              <a href="{{ url_for('serve_book', book_id=mp3file,book_format='mp3') }}"></a>
+            </li>
+          </ul>
+        </div>
+
+      </div>
+
+    </div>
+
+    <script>
+      window.calibre = {
+        filePath: "{{ url_for('static', filename='js/libs/') }}",
+        cssPath: "{{ url_for('static', filename='css/') }}",
+        bookUrl: "{{ url_for('static', filename=mp3file) }}/",
+        bookmarkUrl: "{{ url_for('bookmark', book_id=mp3file, book_format='MP3') }}",
+        bookmark: "{{ bookmark.bookmark_key if bookmark != None }}",
+        useBookmarks: {{ g.user.is_authenticated | tojson }}
+            };
+    </script>
+
+</body>
```

Comment:
My firefox is complaining about thte closing body at this line, there might be a div missing or to much, could you please check this.

Reply:
I'm having a problem with the size of the graphic, on my computer it looks like this:
![bildschirmfoto_2018-06-02_18-04-30](https://user-images.githubusercontent.com/6438895/40878363-5d7c0a54-6690-11e8-83e8-a8e3af71236e.png)
Took me some time to figure out that it is working, and the controller is in the hidden area (where you can't scroll to)